src/app/portlets/content-types/layout/content-types-layout.component.spec.ts

```diff
@@ -75,146 +107,131 @@ describe('ContentTypesLayoutComponent', () => {
         expect(pTabView).not.toBeNull();
     });
 
+    it('should has just one tab', () => {
+        const contentType = fixture.debugElement.query(By.css('.content-type'));
+        const pTabPanels = fixture.debugElement.queryAll(By.css('p-tabPanel'));
+        expect(pTabPanels.length).toBe(1);
+    });
+
     describe('Fields Tab', () => {
         beforeEach(() => {
-            const contentType = fixture.debugElement.query(By.css('.content-type'));
-            this.pTabPanel = contentType.queryAll(By.css('p-tabPanel'))[0];
+            fixture.detectChanges();
+
+            this.pTabPanel = fixture.debugElement.query(By.css('p-tabView .content-type__properties'));
         });
 
         it('should has a field panel', () => {
-            fixture.detectChanges();
             expect(this.pTabPanel).not.toBeNull();
-            expect('Fields Header Tab').toBe(this.pTabPanel.componentInstance.header);
+            expect(this.pTabPanel.componentInstance.header).toBe('Fields Header Tab');
```

Comment:
Just FYI: https://stackoverflow.com/questions/22413009/jasmine-javascript-testing-tobe-vs-toequal

Reply:
Yes in this case really is the same